install_requirements.py

```diff
@@ -42,6 +42,10 @@
 subprocess.check_call(pip_call + ["uninstall", "depthai", "--yes"])
 subprocess.check_call(pip_package_install + ["-r", "requirements.txt"], cwd=scriptDirectory)
 
+# arm version is installed by homebrew in installation script
+if not is_arm:
+    subprocess.check_call(pip_call + ["install", "pyqt5"])
+
```

Comment:
`is_arm` is slightly vague here (and the comment more so)

`platform.machine()` starts with "arm" for:
 - armv6l - ex: Linux (Raspbian) - RPi 1 afaik
 - armv7l - ex Linux (Raspbian) - RPi 4
 - arm64 - ex: Darwin (macOS) - M1 

But also missing aarch64 (ex: Linux (Ubuntu 64bit) - RPi 4, Jetson AFAIK, et al)

Regardless, this shouldn't be needed anymore, as the pyqt5 IS installed by requirements and properly skipped for non platforms that don't carry wheels

Reply:
```suggestion
```